src/home/DappsCarousel.tsx

```diff
@@ -145,10 +139,9 @@ function DappsCarousel({ onSelectDapp }: Props) {
           <Touchable
             key={dapp.id}
             onPress={() => onSelectDapp({ ...dapp, openedFrom: section })}
-            style={styles.dappContainer}
             testID={`${testID}/Dapp`}
           >
-            <>
+            <Card style={styles.card} rounded={true} shadow={Shadow.SoftLight}>
```

Comment:
I think we should put the `Touchable` inside the card like we did for the dapp list: https://github.com/valora-inc/wallet/blob/5ce3b5c541ccdb688ea3165bbaec8b588c6e2338/src/dappsExplorer/DappCard.tsx#L57-L58

This way the ripple effect on Android is contained within it.

Reply:
ah, thanks! i didn't know this, i've updated